src/index.js

```diff
@@ -115,12 +115,13 @@ class Routes {
 }
 
 class Route {
-  constructor ({name, pattern, page = name}) {
+  constructor ({name, pattern, page = name, metadata = {}}) {
```

Comment:
You should call it `meta`, like in Vue Router (https://router.vuejs.org/guide/advanced/meta.html)